core/base/src/TROOT.cxx

```diff
@@ -2136,20 +2124,10 @@ void TROOT::ReadGitInfo()
 
    TString gitinfo = "gitinfo.txt";
    char *filename = 0;
-#ifdef ROOTETCDIR
-   filename = gSystem->ConcatFileName(ROOTETCDIR, gitinfo);
-#else
-   TString etc = gRootDir;
-#ifdef WIN32
-   etc += "\\etc";
-#else
-   etc += "/etc";
-#endif
 #if defined(R__MACOSX) && (TARGET_OS_IPHONE || TARGET_IPHONE_SIMULATOR)
-   // on iOS etc does not exist and gitinfo resides in $ROOTSYS
-   etc = gRootDir;
-#endif
-   filename = gSystem->ConcatFileName(etc, gitinfo);
+   filename = gSystem->ConcatFileName(TROOT::GetRootSys(), gitinfo);
```

Comment:
Does it make sense to move this preprocessor logic into `TROOT::GetEtcDir()`? I.e. have `TROOT::GetEtcDir()` return `TROOT::GetRootSys()` for iphone / simulator?